aaa.py

```diff
@@ -0,0 +1,140 @@
+import binascii
+from tkinter import Tk, filedialog, Label, StringVar, ttk
+
+szukana_wartosc1 = '80 44 00 00 40 44' #ustawienia 640x480
+szukana_wartosc2 = '20 44 00 00 F0 43' #ustawienia 800x600
+szukana_wartosc3 = '48 44 00 00 16 44' #ustawienia 1024x768
+
+nowa_wartosc1 = '80 44 00 00 10 44' # ustawienia 1024x576
+nowa_wartosc2 = 'A0 44 00 00 34 44' # ustawienia 1280x720
+nowa_wartosc3 = 'F0 44 00 00 87 44' # ustawienia 1920x1080
+nowa_wartosc4 = 'C8 44 00 00 61 44' # ustawienia 1600x900
+nowa_wartosc5 = 'AA 44 00 00 40 44' # ustawienia 1360x768
+nowa_wartosc6 = 'F0 44 00 00 B4 44' # ustawienia 1920x1440
+nowa_wartosc7 = '20 45 00 00 B4 44' # ustawienia 2560x1440
+nowa_wartosc8 = '00 B4 44 00 61 44' # ustawienia 1440x900
```

Comment:
takie stałe wartości nazywamy dużymi literami: SZUKANA_WARTOSC1 itd ale ja osobiści bym z tego zrezygnował, bardzo dużą część kodu możesz zamienić na jeden słownik, dokładnie ten mapowanie wartości dodaj sobie go tutaj
```suggestion
mapowanie_wartosci = {
    '80 44 00 00 40 44': '640x480',
    '20 44 00 00 F0 43': '800x600',
    '48 44 00 00 16 44': '1024x768',
    '80 44 00 00 10 44': '1024x576',
    'A0 44 00 00 34 44': '1280x720',
    'F0 44 00 00 87 44': '1920x1080',
    'C8 44 00 00 61 44': '1600x900',
    'AA 44 00 00 40 44': '1360x768',
    'F0 44 00 00 B4 44': '1920x1440',
    '20 45 00 00 B4 44': '2560x1440',
    '00 B4 44 00 61 44': '1440x900',
}
```